packages/twenty-server/src/modules/messaging/message-import-manager/drivers/gmail/services/messaging-gmail-messages-import.service.ts

```diff
@@ -167,6 +167,8 @@ export class MessagingGmailMessagesImportService {
       const messagesToSave = filterEmails(
         messageChannel.handle,
         allMessages,
+        messageChannel.excludeGroupEmails,
+        messageChannel.excludeNonProfessionalEmails,
```

Comment:
I believe the non professional email should still be imported, this only matters for the auto contact creation behavior